src/main/java/task/Task.java

```diff
@@ -0,0 +1,46 @@
+package task;
+
+enum TaskType {
+    TODO, DEADLINE, EVENT
+}
+
+public abstract class Task {
+    private String taskName;
+    private boolean isDone = false;
+    private TaskType taskType;
+
+    //constructor to create a Task object with a task description
```

Comment:
I think javadocs should always start with /** as the comment, rather than //, regardless if its a single or multiline comment?